client/src/components/generic/Page.js

```diff
@@ -1,26 +1,32 @@
 import React, { Fragment, useMemo } from 'react';
 import Grid from '@material-ui/core/Grid';
 import { makeStyles } from '@material-ui/core/styles';
+
 import { AuthContext } from '../../providers';
 import { Header } from './Header';
+import { Footer } from './Footer';
 import { Notifications } from './Notifications';
+import { Routes } from '../../constants';
 
 const useStyles = makeStyles(() => ({
   root: (props) => ({
-    height: props.isAutoHeight ? 'auto' : 'calc(100vh - 82px)',
+    height: 'calc(100vh - 124px)',
     justifyContent: props.centered ? 'center' : 'flex-start',
     alignItems: 'center',
     flexWrap: 'nowrap',
     flexDirection: 'column',
+    display: 'flex',
+    overflow: 'scroll',
```

Comment:
We'll want `overflowY` so we don't get a persistent horizontal scrollbar throughout the site

Reply:
We might want to revisit which element handles overflow, on the participant table we end up with extra scrollbars and most aren't necessary 
![image](https://user-images.githubusercontent.com/71518072/187965303-3993c856-c433-4dc2-ba8c-ed4513d933a5.png)

Reply:
Totally! I'll look more into it today, but I'm also happy to chat on how to handle this best